### PR TITLE
[MOSA-96] - RLP.sol - Use of the "var" keyword is deprecated

### PR DESCRIPTION
**Pull request by abhayks1**
Solution is to use strictly typed variables

### CODE REVIEW

File: contracts/proof/RLP.sol [hunk 1]
```diff
@@ -33,8 +33,8 @@ library RLP {
 
  function next(Iterator memory self) internal pure returns (RLPItem memory subItem) {
      require(hasNext(self));
-     var ptr = self._unsafe_nextPtr;
-     var itemLength = _itemLength(ptr);
+     uint256 ptr = self._unsafe_nextPtr;
```
Comment (deepesh-kn) [inline contracts/proof/RLP.sol]: Do line# 29 needs to change to uint256 ?

Comment (pgev) [reply contracts/proof/RLP.sol]: uint and int are aliases for uint256 and int256, respectively.

Comment (abhayks1) [reply contracts/proof/RLP.sol]: Yes because _itemLength takes ptr as input which is uint256.

File: contracts/proof/RLP.sol [hunk 2]
```diff
@@ -187,9 +189,10 @@ library RLP {
  /// @return Array of RLPItems.
  function toList(RLPItem memory self) internal pure returns (RLPItem[] memory list) {
      require(isList(self));
-     var numItems = items(self);
+     uint numItems;
```
Comment (pgev) [inline contracts/proof/RLP.sol]: Why not declare and define at the same time?

Comment (abhayks1) [reply contracts/proof/RLP.sol]: you are right.
Made the change.

**Comment by benjaminbollen** [comment]
> For Below syntax:
> (uint a, uint b, uint c) = f();
> It will be taken care with below ticket.
> https://ostdotcom.atlassian.net/browse/MOSA-114

for community / public s clarity: this cleaner syntax is possible in solidity v0.4.24 and our internal ticket MOSA-114 is to upgrade solidity to v0.4.24;

**Pull request merged**